crates/sui/src/ptb/ptb_builder/parse_ptb.rs

```diff
@@ -47,6 +47,12 @@ pub struct PTBParser {
     context: PTBContext,
 }
 
+impl Default for PTBParser {
```

Comment:
**sigh** I hate this clippy rule so much...

Nothing to do though, just wanted to mention my dislike of this (IMO stupid) rule that it has.